pkg/loader/testmods/childDirB/go.mod

```diff
@@ -0,0 +1,3 @@
+module sigs.k8s.io/controller-tools/pkg/loader/testmods/childDirB
```

Comment:
Readability nit, can we rename the submodules to s/dir/submodule, i.E. here childSubmoduleB?

Reply:
Oh, sure. Makes total sense.

Reply:
Actually, wouldn't that be slightly confusing it the mod name isn't consistent with the file structure? And while I could rename the dir names, that leaves the dir names that *aren't* modules...

Reply:
right but the dirs that aren't modules are just dirs so that should be fine?

Reply:
So you'd have `submodA` and `childdirB`?

Reply:
yeah, exactly

Reply:
I mean, my eye might twitch at the inconsistency, but I can live with it 😃